lib/nimble_template/hex/package_mock.ex

```diff
@@ -0,0 +1,14 @@
+defmodule NimbleTemplate.Hex.PackageMock do
+  @moduledoc false
+
+  @behaviour NimbleTemplate.Hex.PackageBehaviour
+
+  alias NimbleTemplate.Exception.MockHexPackageRequiredException
+
+  def get_latest_version(package) do
+    raise(MockHexPackageRequiredException,
+      message:
+        "Mock Hex Package with `@describetag mock_latest_package_versions: [{#{package}, :version}]` required"
```

Comment:
How about?
```suggestion
        "Requires to mock Hex package with `@describetag mock_latest_package_versions: [{#{package}, :version}]`"
```

Reply:
updated in https://github.com/nimblehq/elixir-templates/pull/284/commits/0a89e641f577bdf2e4e0b1545e3dea1f7567369f :)